### ISSUE TITLE
Error building the tutorial app

### ISSUE THREAD

**Issue by FabrizioDonnarumma** [dotnet/maui#22692]
Problem encountered on https://dotnet.microsoft.com/en-us/learn/maui/first-app-tutorial/run
Operating System: windows

Provide details about the problem you're experiencing. Include your operating system version, exact error message, code sample, and anything else that is relevant.

Hello, 

I run into the issue below when building the project. I did not make any modifications to it, simply created the MAUI project and built it.
The building process took ~ 90 seconds, and then this error popped up

```
Severity	Code	Description	Project	File	Line	Suppression State
Error (active)	MSB4018	The "CreateAppHost" task failed unexpectedly.
System.IO.IOException: The requested operation cannot be performed on a file with a user-mapped section open.

   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.MemoryMappedFiles.MemoryMappedFile.CreateCore(SafeFileHandle fileHandle, String mapName, HandleInheritability inheritability, MemoryMappedFileSecurity memoryMappedFileSecurity, MemoryMappedFileAccess access, MemoryMappedFileOptions options, Int64 capacity)
   at System.IO.MemoryMappedFiles.MemoryMappedFile.CreateFromFile(FileStream fileStream, String mapName, Int64 capacity, MemoryMappedFileAccess access, MemoryMappedFileSecurity memoryMappedFileSecurity, HandleInheritability inheritability, Boolean leaveOpen)
   at Microsoft.NET.HostModel.ResourceUpdater.Update()
   at Microsoft.NET.HostModel.AppHost.HostWriter.<>c__DisplayClass2_0.<CreateAppHost>b__1()
   at Microsoft.NET.HostModel.RetryUtil.RetryOnIOError(Action func)
   at Microsoft.NET.HostModel.AppHost.HostWriter.CreateAppHost(String appHostSourceFilePath, String appHostDestinationFilePath, String appBinaryFilePath, Boolean windowsGraphicalUserInterface, String assemblyToCopyResourcesFrom, Boolean enableMacOSCodeSign)
   at Microsoft.NET.Build.Tasks.CreateAppHost.ExecuteCore()
   at Microsoft.NET.Build.Tasks.TaskBase.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	MyFirstMauiApp (net8.0-windows10.0.19041.0)	C:\Program Files\dotnet\sdk\8.0.300\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.targets	709	
```

My machine runs 
Symantec Endpoint Protection. I tried to build with it deactivated, but I got the same error. I also tried running VS with admin privileges (although I launch it from an admin account), and I get the same error. 
I have used VS for several winform and console apps, therefore I know that it can write to the location I chose.
I wonder if Windows is blocking the app because it does not recognize it?

**Comment by moljac** [comment]
Thanks for the feedback.

> I tried to build with it deactivated, but I got the same error. I also tried running VS with admin privileges (although I launch it from an admin account), and I get the same error.

Those were good attempts. I would try with:

1. nuking `bin/` and `obj/` folders
2. closing all VS and VSCode instances that might be holding that project opened
3. fix VS and Workloads installation[s]
4. reboot 


> I have used VS for several winform and console apps, therefore I know that it can write to the location I chose.

MAUI is different. Much more complicated.

> I wonder if Windows is blocking the app because it does not recognize it?

I don't think so, but I don't have enough info to be sure.

**Comment by FabrizioDonnarumma** [comment]
> Thanks for the feedback.
> 
> > I tried to build with it deactivated, but I got the same error. I also tried running VS with admin privileges (although I launch it from an admin account), and I get the same error.
> 
> Those were good attempts. I would try with:
> 
> 1. nuking `bin/` and `obj/` folders
> 2. closing all VS and VSCode instances that might be holding that project opened
> 3. fix VS and Workloads installation[s]
> 4. reboot
> 
> > I have used VS for several winform and console apps, therefore I know that it can write to the location I chose.
> 
> MAUI is different. Much more complicated.
> 
> > I wonder if Windows is blocking the app because it does not recognize it?
> 
> I don't think so, but I don't have enough info to be sure.

Hello, 

Thanks for the reply. 
I tried all the steps you proposed, but I get the same error. 
In addition, I tried to create to build the same project on a different machine that runs VS 2022 (same version, same components and the one giving me the error). The main differences between the two workstations is that the one causing this error uses Win 10.0.19045 Build 19045), while the other uses win Version 10.0.22631 Build 22631.
Also, the win 11 workstation does not have Symantec Endpoint Protection.

On that machine, I encounter a different problem: the build starts, and never ends. IT starts 5 different instances of MSBuild.exe and they all seem to be sitting there doing nothing but for one, which uses 2-3% of the processor every few seconds. At the same time, no new files are created in the bin folder other than the subfolders.

ON the step before the building, there is a note at the end that says "Visual Studio creates the new project. If you get a Windows Security Alert warning you about the firewall blocking some features, select the Allow access button."
I did not get any such warning. Could it be my firewall is blocking the building process?

Also, the project is created on a google drive folder. Could that be part of the problem too?
